### DISCUSSION TITLE
Ability to print trees (directories)

### DISCUSSION THREAD

**Discussion by ssbarnea** [Textualize/rich#412] [Ideas]
I am currently using another small library to print a simple three but it could prove useful if rich would also be able to do something like:
```
└── folder
    ├── foo
    ├── bar
    └── destroy
```

**Comment by willmcgugan** [comment]
Yes, I was thinking about porting something like https://docs.pyfilesystem.org/en/latest/guide.html#tree-printing to Rich. Post it as an issue with the "enhancement" label if you want to track it.

**Comment by ssbarnea** [comment]
For the moment I decided to re-use the YAML printing and avoid the need to render a real tree. See https://github.com/ansible-community/molecule/pull/2920 -- it looks tree enough for me and one less dependency is always something to be appreciated.
